bower.json

```diff
@@ -20,6 +20,7 @@
     "moment-timezone": "0.5.3",
     "patternfly": "3.43.0",
     "jquery": "3.2.1",
+    "jquery-qrcode": "*",
```

Comment:
Since this will affect the shippable code, pin it to an exact version.  We are more flexible with `package.json` since it is our build tools, but anything in `bower.json` should be explicit.  We verify our build via CI, so any difference in version will fail tests.